L1Trigger/L1TTrackMatch/BuildFile.xml

```diff
@@ -0,0 +1,49 @@
+<use   name="FWCore/Framework"/>
+<use   name="FWCore/PluginManager"/>
+<use   name="FWCore/ParameterSet"/>
+<use   name="FWCore/Utilities"/>
+<use   name="CondFormats/DataRecord"/>
+<use   name="CondFormats/L1TObjects"/>
+<use   name="Geometry/Records"/>
+<use   name="Geometry/CommonDetUnit"/>
+<use   name="Geometry/CommonTopologies"/>
+<use   name="Geometry/DTGeometry"/>
+<use   name="Geometry/TrackerGeometryBuilder"/>
+<use   name="DataFormats/Common"/>
+<use   name="DataFormats/DetId"/>
+<use   name="DataFormats/GeometryVector"/>
+<use   name="DataFormats/GeometrySurface"/>
+<use   name="DataFormats/Phase2TrackerDigi"/>
+<use   name="DataFormats/TrackReco"/>
+<use   name="DataFormats/VertexReco"/>
+<use   name="TrackingTools/TrackAssociator"/>
+<use   name="PhysicsTools/CandUtils"/>
+<use   name="FWCore/MessageLogger"/>
+<use   name="DataFormats/L1Trigger"/>
+<use   name="DataFormats/L1TrackTrigger"/>
+<use   name="DataFormats/L1TCorrelator"/>
+<use   name="DataFormats/HepMCCandidate"/>
+<use   name="SimDataFormats/Vertex"/>
+<use   name="SimDataFormats/TrackingHit"/>
+<use   name="SimDataFormats/GeneratorProducts"/>
+<use   name="TrackingTools/Records"/>
+<use   name="TrackingTools/PatternTools"/>
+<use   name="TrackingTools/TrajectoryState"/>
+<use   name="TrackingTools/GeomPropagators"/>
+<use   name="TrackingTools/TransientTrack"/>
+<use   name="TrackingTools/TrajectoryParametrization"/>
+<use   name="RecoTracker/TrackProducer"/>
+<use   name="L1Trigger/L1TMuon"/>
+<use   name="clhep"/>
+<use   name="xerces-c"/>
+<use   name="boost"/>
+<use   name="root"/>
+<use   name="rootrflx"/>
+<use   name="SimDataFormats/Track"/>
+<use   name="DataFormats/SiPixelDetId"/>
+<use   name="DataFormats/Math"/>
+<use   name="CommonTools/BaseParticlePropagator"/>
+<export>
+  <flags   SEALPLUGIN="1"/>
+  <lib   name="1"/>
```

Comment:
perhaps we don't need both a src and a plugins directory full of plugins?

otherwise, the actual dependencies of this package appear to be very small fraction of those listed in the Buidlfile. A cleanup would be useful.

Reply:
@davidlange6 
Cleanup is done. 

There are only these 2 in src 
```
L1Trigger/Phase2L1ParticleFlow/src/BitwisePFAlgo.cc
L1Trigger/Phase2L1ParticleFlow/src/COEFile.cc
```